_shows/19_20/freshers_fringe.md

```diff
@@ -0,0 +1,34 @@
+---
+title: 'Freshers\' Fringe'
```

Comment:
Seems upset with this line, try `title: "Freshers' Fringe"`

You can test if the YAML section is valid by pasting into http://www.yamllint.com/